compiler_opt/es/blackbox_evaluator.py

```diff
@@ -42,7 +42,7 @@ def get_results(
     raise NotImplementedError()
 
   @abc.abstractmethod
-  def set_baseline(self) -> None:
+  def set_baseline(self, pool: FixedWorkerPool) -> None:
```

Comment:
should the pool be passed in the ctor? it's not like you'd keep changing the pool every time you called set_baseline, or?